cli/dcoscli/marathon/main.py

```diff
@@ -279,13 +279,17 @@ class MarathonSubcommand(object):
     :param create_marathon_client: a callable that returns an instance of
                                    marathon.Client
     :type create_marathon_client: collections.abc.Callable
+    :param event_emitter: sink for output events, e.g. messages to the console
+    :type event_emitter: emitting.Emitter
```

Comment:
I don't think we should have this here, we want all our emitting in the cli to stdout to be consistent.